builtin/providers/aws/resource_aws_iam_policy.go

```diff
@@ -81,7 +86,29 @@ func resourceAwsIamPolicyRead(d *schema.ResourceData, meta interface{}) error {
 		return fmt.Errorf("Error reading IAM policy %s: %s", d.Id(), err)
 	}
 
-	return readIamPolicy(d, response.Policy)
+	getPolicyVersionRequest := &iam.GetPolicyVersionInput{
```

Comment:
Why are we looking for a specific policy version? This is a new code path that wasn't in use before